src/cswinrt/code_writers.h

```diff
@@ -2513,43 +2511,6 @@ Marshal.Release(inner);
             {
                 write_composable_constructors(w, factory.type, type, factory.visible ? "public"sv : "protected"sv);
             }
-            else if (factory.statics)
```

Comment:
We probably can't stop generating these functions due to back compat.  When we generate the Windows SDK projection for instance, the backcompat check there will break.